<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RankWindow</class>
 <widget class="QWidget" name="RankWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="RankTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>521</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_rank">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Adobe Devanagari&quot;;</string>
   </property>
   <property name="text">
    <string>闯关者排行</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_switch">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>切换</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_logout">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>490</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>注销</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BS_u">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>按用户名排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BS_s">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>490</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>按关数排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BS_l">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>490</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>按等级排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BS_t">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>490</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>按时间排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_game">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>开始单机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_check">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>查询用户</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rank_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Adobe Devanagari&quot;;</string>
   </property>
   <property name="text">
    <string>/查询结果</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_u">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_l">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>等  级</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_t">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>时      间</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_s">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>关卡/出题数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_online">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>上线</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_offline">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>下线</string>
   </property>
  </widget>
  <widget class="QPushButton" name="B_online_check">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>170</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>查询在线用户</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="L_challenge">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>280</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="B_challenge">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>320</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>向⬆️️发起挑战</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
